server/app/services/export/PdfExporter.java

```diff
@@ -19,9 +20,12 @@
 import java.time.ZoneId;
 import javax.inject.Inject;
 import models.Application;
+import services.Path;
 import services.applicant.AnswerData;
 import services.applicant.ApplicantService;
 import services.applicant.ReadOnlyApplicantProgramService;
+import services.applicant.question.Scalar;
+import services.question.types.QuestionType;
 
 /** PdfExporter is meant to generate PDF files. The functionality is not fully implemented yet. */
 public class PdfExporter {
```

Comment:
Not blocking: Would you mind marking this as `final` to make the decision to allow subclassing explicit rather than implicit?

Reply:
Sure